httpcore5/src/main/java/org/apache/hc/core5/http/config/Http1Config.java

```diff
@@ -222,6 +235,14 @@ public Builder setInitialWindowSize(final int initialWindowSize) {
             return this;
         }
 
+        /**
+         * @since 5.4
```

Comment:
I think you need a real Javadoc description that explains what this toggle does.